newscoop/src/Newscoop/NewscoopBundle/Controller/CommentsController.php

```diff
@@ -36,7 +36,7 @@ public function indexAction(Request $request)
         $blogService = $this->get('blog');
         $user = $userService->getCurrentUser();
 
-        if ($blogService->isBlogger($user)) {
```

Comment:
what exaclty it would break? It will don't check only if user is blogger but if user have permission to manage comemments (blogers don't have it).